app/quest-boost/[boostId]/page.tsx

```diff
@@ -0,0 +1,195 @@
+"use client";
+
+import React, { useContext, useEffect, useState } from "react";
+import styles from "../../../styles/questboost.module.css";
+import { Call, CallDetails } from "starknet";
+import {
+  getBoostById,
+  getQuestBoostClaimParams,
+  getQuestParticipants,
+  getQuestsInBoost,
+  updateQuestBoostClaimStatus,
+} from "../../../services/apiService";
+import Quest from "../../../components/quests/quest";
+import { useRouter } from "next/navigation";
+import { QuestDocument } from "../../../types/backTypes";
+import Timer from "../../../components/quests/timer";
+import { useAccount, useContractWrite } from "@starknet-react/core";
+import Button from "../../../components/UI/button";
+import boostContractCalls from "../../../utils/calldata/boost_contract_claim";
+import { StarknetIdJsContext } from "../../../context/StarknetIdJsProvider";
+import { useNotificationManager } from "../../../hooks/useNotificationManager";
+import {
+  NotificationType,
+  TransactionType,
+} from "../../../constants/notifications";
+import { hexToDecimal } from "../../../utils/feltService";
+import { CDNImage } from "../../../components/cdn/image";
+
+type BoostQuestPageProps = {
+  params: {
+    boostId: string;
+  };
+};
+
+export default function Page({ params }: BoostQuestPageProps) {
+  const router = useRouter();
+  const { address } = useAccount();
+  const { boostId } = params;
+  const [quests, setQuests] = useState([] as QuestDocument[]);
+  const [boost, setBoost] = useState({} as Boost);
+  const [participants, setParticipants] = useState<number>();
+  const [calls, setCalls] = useState<CallDetails>({} as CallDetails);
+  const [sign, setSign] = useState<string[]>([]); // [r, s]
```

Comment:
Don't understand the comment

Reply:
This is (r,s) components of the ECDSA signature which we get from api

Basically we save these values in an array since they are used together. The values are saved in the format of [r,s] to prevent confusion if someone needs to modify it or wants to understand the format

Reply:
So I would say to type it like so because the array of strings can represent something else. 

this would work `type Signature = Array<string>(2)`